### PR TITLE
Fixed the behavior of plugin. Now it checks for files that has .java extension.

### PR DESCRIPTION
**Pull request by alexander-aleksandrov**

### CODE REVIEW

**Comment by alexander-aleksandrov** [comment]
@dmdashenkov PTAL

File: javadoc-checker/plugin/src/main/java/org/spine3/tools/javadoc/fqnchecker/FqnCheckPlugin.java [hunk 1]
```diff
@@ -137,16 +138,23 @@ public FileVisitResult visitFileFailed(Path file, IOException exc) throws IOExce
 
     static void check(Path file) throws InvalidFqnUsageException {
         final String content;
-        final byte[] rawContent;
+        if (!file.toString().endsWith(".java")){
```
Comment (dmdashenkov) [inline javadoc-checker/plugin/src/main/java/org/spine3/tools/javadoc/fqnchecker/FqnCheckPlugin.java]: Please add tests for this rule.

File: javadoc-checker/plugin/src/main/java/org/spine3/tools/javadoc/fqnchecker/FqnCheckPlugin.java [hunk 2]
```diff
@@ -137,16 +138,23 @@ public FileVisitResult visitFileFailed(Path file, IOException exc) throws IOExce
...
+            return;
+        }
         try {
             content = Files.readAllLines(file, StandardCharsets.UTF_8).toString();
         } catch (IOException e) {
             throw new IllegalStateException("Cannot read the contents of the file: " + file, e);
         }
         final Optional<InvalidFqnUsage> checkResult = check(content);
```
Comment (dmdashenkov) [inline javadoc-checker/plugin/src/main/java/org/spine3/tools/javadoc/fqnchecker/FqnCheckPlugin.java]: `final Optional<InvalidFqnUsage> styleViolation`.

File: javadoc-checker/plugin/src/main/resources/WrongFQNformat.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package org.spine3.tools.javadoc.fqnchecker;
+public class WrongFQNformat {
+    /**{@link org.spine3.server.event.EventBus}*/
+    /**{@link org.spine3.server.event.EventBus                }*/
```
Comment (dmdashenkov) [inline javadoc-checker/plugin/src/main/resources/WrongFQNformat.java]: Which of those two cases is invalid?

**Comment by dmdashenkov** [comment]
@alexander-aleksandrov, please wait until I finish the checks.

File: javadoc-checker/plugin/src/main/resources/WrongFQNformat.java (hunk 3 above)
Comment (alexander-aleksandrov) [inline javadoc-checker/plugin/src/main/resources/WrongFQNformat.java]: Need both as discussed.

File: javadoc-checker/plugin/src/main/java/org/spine3/tools/javadoc/fqnchecker/FqnCheckPlugin.java (hunk 2 above)
Comment (alexander-aleksandrov) [inline javadoc-checker/plugin/src/main/java/org/spine3/tools/javadoc/fqnchecker/FqnCheckPlugin.java]: Done

File: javadoc-checker/plugin/src/main/java/org/spine3/tools/javadoc/fqnchecker/FqnCheckPlugin.java (hunk 1 above)
Comment (alexander-aleksandrov) [inline javadoc-checker/plugin/src/main/java/org/spine3/tools/javadoc/fqnchecker/FqnCheckPlugin.java]: It is present in resources as hidden file with broken FQN links inside.

**Comment by dmdashenkov** [comment]
@alexander-aleksandrov, LGTM.

**Pull request merged**